src/main/java/com/amazon/janusgraph/batch/BatchCommand.java

```diff
@@ -0,0 +1,54 @@
+package com.amazon.janusgraph.batch;
```

Comment:
You did not add a new tutorial, so the code that supports your minimal reproducing test for the issue reported in Stack Overflow belongs in test.